assets/js/app.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Welcome to your app's main JavaScript file!
+ *
+ * We recommend including the built version of this JavaScript file
+ * (and its CSS file) in your base layout (base.html.twig).
+ */
+
+// any CSS you require will output into a single css file (app.css in this case)
+require('../css/app.css');
+
+// Need jQuery? Install it with "yarn add jquery", then uncomment to require it.
+// const $ = require('jquery');
+
+import $ from 'jquery';
```

Comment:
Should it not be imported like mentioned above?

`const $ = require('jquery');`

Reply:
It could be required like that. In the end I decided to use the dynamic import included in the alternate syntax based on the ecmascript 6 modules.

I removed the comments with commit 0d30f19